Sources/Sentry/Public/PrivateSentrySDKOnly.h

```diff
@@ -35,6 +41,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (NSArray<SentryDebugMeta *> *)getDebugImages;
 
+@property (class, nullable, nonatomic, copy)
+    SentryOnAppStartMeasurementAvailable onAppStartMeasurementAvailable;
+
+@property (class, nullable, nonatomic, readonly) SentryAppStartMeasurement *appStartMeasurement;
+
+/**
+ * If enabled, the SDK won't send the app start measurement with the first transaction. Instead, if
+ * enableAutoPerformanceTracking is enabled, the SDK measures the app start and then calls
```

Comment:
In case of Unity where we init the SDK early (in `main.mm`) we just don't flip this flag?

Reply:
Yes, we could do that. We can also change it and add another mode just for Unity. I guess it will take a while until Unity is ready for that, so things could change a bit.